src/components/ControlItem.tsx

```diff
@@ -28,7 +28,8 @@ const createStyles = (colors: any, isHovered: boolean, isPressing: boolean) =>
       flexDirection: 'row',
       gap: 16,
       width: 360,
-      height: 90,
+      height: '50%',
+      minHeight: 110,
```

Comment:
```suggestion
      minHeight: 90,
```
I don't think you need the 50% height.